project/SparkBuild.scala

```diff
@@ -621,6 +621,7 @@ object KubernetesIntegrationTests {
 object DependencyOverrides {
   lazy val settings = Seq(
     dependencyOverrides += "com.google.guava" % "guava" % "14.0.1",
+    dependencyOverrides += "xerces" % "xercesImpl" % "2.12.0",
```

Comment:
SBT seems picking xml-apis and xerces fine when hadoop-2.7 profile is used:

```
[info]   | | | | +-xerces:xercesImpl:2.12.0
[info]   | | | | | +-xml-apis:xml-apis:1.4.01
```

Reply:
and .. not picked when hadoop-3.2 profile is used. If we concern about including `xml-apis`, we can just revert `xerces` too. I will leave it to you @srowen - I guess you have better insights on this.

Reply:
BTW, for `hadoop-3.2`, `xercesImpl` itself is not used, isn't it?

Reply:
+1, Ideally yes, I think so.